assets/src/blocks/Covers/CampaignCovers.js

```diff
@@ -14,37 +14,41 @@ export const CampaignCovers = ({
 }) => (
   <div className='covers'>
     {covers.map((cover, index) => {
-      const { href, image, alt_text, name, src_set } = cover;
+      const { image, alt_text } = cover;
       const hideCover = !isCarouselLayout && !!initialRowsLimit && index >= row * amountOfCoversPerRow;
 
       if (hideCover) {
         return null;
       }
 
-      const campaignLink = inEditor ? null : href;
+      const link = cover.link || cover.href;
+      const srcset = cover.srcset || cover.src_set;
+      const tag = cover.tag || cover.name;
```

Comment:
Shouldn't this be called `title`? :thinking:

Reply:
I was hesitating about that, but since it's not really a title I decided to give it a more accurate name... But we could indeed use `title` instead

Reply:
That makes sense, but then we again have the problem of the block not receiving the expected data when switching between styles.

That's why we shouldn't use the block styles to control what data is being shown, this API is only intended by WP to put the class name there. I guess this means rethinking how the block works, or converting the `registerBlockStyles` thing to our own component that can actually work with our logic.

Reply:
Yes it makes sense, just added a commit with this change!